Audio-Converter-GUI/new.py

```diff
@@ -0,0 +1,116 @@
+import tkinter as tk
```

Comment:
- If the video file is not found in the directory, it is still showing the option to convert into audio and if I press any of the options, errors are raised:
![image](https://user-images.githubusercontent.com/43691873/111123548-78e7a380-8595-11eb-8d05-2c358fedfa98.png)

- After this error, if put the right file, it is not resetting the message shown though the conversion takes place:
![image](https://user-images.githubusercontent.com/43691873/111123866-d8de4a00-8595-11eb-9110-ed4b5b18c05d.png)

- If the wrong file is given as input in the first place, and then right file is given as input, no success message is displayed in the application 

- Display a progress bar so that user is assured that something is happening